proprietary/tokens/build.js

```diff
@@ -3,7 +3,7 @@ const StyleDictionary = require('style-dictionary')
 const modes = ['compact']
 
 function generateSharedConfig(mode) {
-  const name = mode ? `${mode}` : 'index'
+  const name = mode || 'index'
```

Comment:
I think we can use the null coalescing operator more often – this is one example where our intent is to check for a value that is not defined, not a falsy one ([SO](https://stackoverflow.com/questions/61480993/when-should-i-use-nullish-coalescing-vs-logical-or)).

```suggestion
  const name = mode ?? 'index'
```

E.g. an unfortunate mode of `''` would result in a class of `.amsterdam-theme--`.

Reply:
> E.g. an unfortunate mode of '' would result in a class of .amsterdam-theme--.

But this is something we don't want, right? I think you would get this using the nullish coalescing operator, but not with the OR operator. I don't think it's necessarily a bad idea to not allow falsy values as mode names...

Tangentially related: maybe we should convert this to Typescript, to prevent possible issues like these?

Reply:
The current approach will work as well; let’s discuss later.